libs/agora/teams/src/lib/team-list/team-list.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="team-list">
```

Comment:
About the use of "team-list" here: this support the idea of naming the OAS schema "TeamList" instead of "Teams".

Reply:
will do

Reply:
makes sense, legacy repo hasn't been consistent on this.